### DISCUSSION TITLE
unzipped file names has a dot-like notation

### DISCUSSION THREAD

**Discussion by tjhunkin-inhance** [spatie/laravel-backup#1748] [General] [answered]
unzipped file names from the backup have a dot-like notation, is it supposed to be like that? How do I keep the backup folder structure the same as my application structure, making it easier to override if I need to restore it?

EDIT: this seems to only be an issue on Windows

**Comment by parallels999** [comment, marked as answer]
>It's not compatible with Windows. 

https://github.com/spatie/laravel-backup/blob/3734a6124b19be7f92333299c33fed5ed650f60d/docs/requirements.md?plain=1#L6
